recipe-page.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE>
+<html>
+  <head>
+    <link href='http://fonts.googleapis.com/css?family=Abel|Yellowtail|Droid+Serif' rel='stylesheet' type='text/css'>
+    <title>Delicious Recipes</title>
+    <link href="style.css" type="text/css" rel="stylesheet" />
+    <link href="style-grid.css" type="text/css" rel="stylesheet" />
+  </head>
+  <div class='bodyContainer'>
```

Comment:
I don't think your grid is working. All of your grid classes don't have underscores and this one isn't even a grid class.